lint-rules-android-lint/src/test/java/com/vanniktech/lintrules/android/AnnotationOrderDetectorTest.kt

```diff
@@ -226,6 +226,30 @@ class AnnotationOrderDetectorTest {
             |+   @Binds @Singleton @NonNull void test() { }
             |""".trimMargin())
   }
+  
+  @Test fun singletonBeforeNonNullBeforeCustom() {
```

Comment:
Kept this one. Not strictly about the issue but verifies that the `lintFix` puts the custom annotation to the end.